src/main/java/core/db/UserConfigurationTable.java

```diff
@@ -117,6 +118,24 @@ private String getStringValue(String key) {
 		return value;
 	}
 
+	private HashMap<String, String> getAllStringValues() {
+		final StringBuffer sql = new StringBuffer(100);
+		sql.append("SELECT * FROM ");
+		sql.append(getTableName());
+
+		HashMap<String, String> map = new HashMap<>();
+		final ResultSet rs = adapter.executeQuery(sql.toString());
+		try {
+			while (rs.next()) {
+				map.put(rs.getString("CONFIG_KEY"), rs.getString("CONFIG_VALUE"));
```

Comment:
Some of the values are not strings, can that cause issues in other places?

Reply:
I think I kept the same logic as before ... it was converting everything to string ... but might worth to check